ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -197,6 +259,42 @@ export default class StatusDropdown extends React.Component {
             }
           />
         )}
+        <FileBugModal
+          showModal={showFileBugModal}
+          toggle={() => this.toggle('showFileBugModal')}
+          updateAndClose={(event, inputValue) =>
+            this.fileBugAndClose(
+              event,
+              {
+                bug_number: parseInt(inputValue, 10),
+                issue_tracker: issueTrackers.find(
+                  (item) => item.text === selectedValue,
+                ).id,
+              },
+              'showFileBugModal',
+            )
+          }
+          header="File Bug for"
+          title="Bug Number"
+          submitButtonText="File Bug"
+          dropdownOption={
+            <Col>
+              <Label for="issueTrackerSelector">Select Bug Tracker</Label>
```

Comment:
There's a pretty big problem with the alternate issue tracker we have, namely Github.
It was added mostly for linking perf improvements to the Github PRs, back in the Servo days.
It wasn't designed to link regressions on Github (it was a half-legged implementation).

My suggestion (though I'm open for other ideas): when filing bugs, we can simply eliminate the dropdown. Implicitly, the modal will target only the Bugzilla issue tracker.
We'll be left with only the input text. Ideally, the user should quickly see this field is optional.

Reply:
I removed the options, but I left the dropdown inplace. The default value is Bugzilla and there's no other option to select. I tried removing the dropdown at all but the modal look pretty odd and incomplete.
![Screenshot from 2020-12-04 11-09-18](https://user-images.githubusercontent.com/47977085/101144578-2f47fe00-3621-11eb-84a8-af53d0afa8f0.png)

Reply:
That dropdown feels buggy. We already have popups identical to this one, so I'm not sure why we'd do things differently.